eng/helix/vstest/runtests.cmd

```diff
@@ -1,3 +1,6 @@
+REM Disable "!Foo!" expansions because they break the filter syntax
+setlocal disableextensions
```

Comment:
cc @HaoK 	

Even with the planned change to ignore the exit code from helix altogether I'd like to get this change in in some form so that when we do start paying attention to the exit code, we don't start getting known-flaky tests causing failures.

Reply:
(Still working on Unix scripts btw)

Reply:
Yup, ignoring the exit code is hopefully just a short/medium term thing that gives us some time to get flaky in place

Reply:
maybe consider passing in the filters to each script (assuming they are the same in the sh version)

Reply:
Yeah, they would be the same. I'll look at that. There can be some shell escaping problems with the syntax so it might be tricky, but I'll give it a shot.